app/support/version.rb

```diff
@@ -12,7 +12,7 @@ module Version
   # Bespoke error class to highlight version errors
   class Error < StandardError
     def message
-      'Not a valid version'
+      'Unsupported or invalid version'
     end
```

Comment:
Version::Error is a generic error class in app/support/version so we would need to overload it or extend it to give a custom error. But I agree that it would be useful to give a more specific error. Ill take a look. 
Good point about the UI - I assumed it would just display the error like normal but it doesn't error handle this request so it shows the user a json object. Ill add some error handling there 👍

Reply:
Updated it to take a custom message overload and made the sample sheet error more descriptive